supplier_api_tester/supplier_api_tester/cli.py

```diff
@@ -17,25 +17,20 @@ def print_title(title):
 @click.option('-k', '--api-key', required=True, prompt='API Key', type=str)
 @click.option('-p', '--product-id', required=True, prompt='Product ID', type=str)
 @click.option('-t', '--timeslots', is_flag=True, default=False, help='Use timeslots')
-@click.option('-a', '--availability', is_flag=True, default=False, help='Run availability tests')
-@click.option('-r', '--reservation', is_flag=True, default=False, help='Run reservation tests')
-@click.option('-b', '--booking', is_flag=True, default=False, help='Run booking tests')
-def supplier_tester(url, api_key, product_id, timeslots, availability, reservation, booking):
+@click.option('-a', '--availability', is_flag=True, default=True, help='Run availability tests')
```

Comment:
The default value should be `False` because if everything is `False` then we're running a full test. After this change to run only availability tests we would have to switch eveything else to `False`.